pkg/server/version.go

```diff
@@ -17,14 +17,35 @@ limitations under the License.
 package server
 
 import (
-	"errors"
+	"fmt"
 
+	"github.com/golang/protobuf/ptypes/empty"
 	"golang.org/x/net/context"
 
 	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1"
 )
 
+const (
+	containerName        = "containerd"
+	containerdAPIVersion = "0.0.0"
+	containerdVersion    = "0.0.0"
+	// kubeAPIVersion is the api version of kubernetes.
+	kubeAPIVersion = "0.1.0"
+)
+
 // Version returns the runtime name, runtime version and runtime API version.
 func (c *criContainerdService) Version(ctx context.Context, r *runtime.VersionRequest) (*runtime.VersionResponse, error) {
-	return nil, errors.New("not implemented")
+	_, err := c.versionService.Version(ctx, &empty.Empty{})
+	if err != nil {
+		return nil, fmt.Errorf("failed to get containerd version: %v", err)
+	}
+	return &runtime.VersionResponse{
+		Version:     kubeAPIVersion,
+		RuntimeName: containerName,
+		// Containerd doesn't return semver because of a bug.
+		// TODO(random-liu): Replace this with version from containerd.
+		RuntimeVersion: containerdVersion,
+		// Containerd doesn't have an api version now.
```

Comment:
containerd won't use date to set version I think.

They should expose api version, but I think it's to early to them to talk about api version now, given that they are changing api everyday before GA.